src/containers/pages/FocusInvestigation/single/tests/index.test.tsx

```diff
@@ -49,15 +49,29 @@ describe('containers/pages/SingleFI', () => {
         path: `${FI_SINGLE_URL}/:id`,
         url: `${FI_SINGLE_URL}/13`,
       },
-      planById: fixtures.plan1,
+      planById: null,
```

Comment:
@p-netm why this change?

Reply:
To increase coverage. for this test, we would also be testing that the loading component was shown. Later on in the same testCase i replace the null value with a valid planId.